Source/Projections/Store/Builders/ProjectionsBuilder.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Execution;
+using Dolittle.SDK.Projections.Store.Converters;
+using Dolittle.SDK.Services;
+using Dolittle.SDK.Tenancy;
+using Microsoft.Extensions.Logging;
+using ExecutionContext = Dolittle.SDK.Execution.ExecutionContext;
+
+namespace Dolittle.SDK.Projections.Store.Builders
+{
+    /// <summary>
+    /// Represents a builder for building <see cref="Projections"/>.
+    /// </summary>
+    public class ProjectionsBuilder
```

Comment:
Let's rename this so we have less collisions in the Client etc.
```suggestion
    public class ProjectionStoreBuilder
```